src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -2,19 +2,29 @@
 
 # useMediaStreamMetrics
 
-The `useMediaStreamMetrics` hook returns the available audio, video and bandwidth metrics for the current meeting session. Video metrics contains stream metrics for all attendeeIds.
+The `useMediaStreamMetrics` hook returns:
+
+- common metrics for audio stream
+- common metrics for video stream of all attendees
+- original [RTCStatsReport](https://developer.mozilla.org/en-US/docs/Web/API/RTCStatsReport) returned by [RTCPeerConnection.getStats()](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getStats)
 
 This hook re-renders frequently. You should avoid using it high in your app tree, and ideally use it in your leaf components.
 
 ### Return Value
 
-```javascript
+```ts
 {
   audioPacketsSentFractionLossPercent: number | null;
   audioPacketsReceivedFractionLossPercent: number | null;
+  audioSpeakerDelayMs: number | null;
+  audioUpstreamRoundTripTimeMs: number | null;
+  audioUpstreamJitterMs: number | null;
+  audioDownstreamJitterMs: number | null;
+  currentRoundTripTimeMs: number | null;
   availableOutgoingBandwidth: number | null;
   availableIncomingBandwidth: number | null;
-  videoStreamMetrics: { [attendeeId: string]: { [ssrc: string]: {[key: string]: number} } }
+  rtcStatsReport: RTCStatsReport | null;
```

Comment:
Did not knew `metricsDidReceive` now gives the entire stats report. Was there a discussion around the redundancy here, are the rest of the stats already part of the rtsStatsReport? [Just checking for understanding]

Reply:
>Did not knew metricsDidReceive now gives the entire stats report. 

Yeah I added this to js sdk in my V3 legacy metrics migration task as Trung and Henri suggested.

>Was there a discussion around the redundancy here

Since we still make only one `getStats` call, so no performance issue here. Discussed with Trung about this in JS SDK. However for React SDK I just brought this up in the standup early this week. And we agreed that it's good to have this, so we don't have to update this hook every time when customer requests some specific metrics.

>are the rest of the stats already part of the rtsStatsReport


All these metrics are originally from this `rtcStatsReport`, but for the rest metrics in this hook, most of them are processed, e.g, convert from second to millisecond, made some calculation. The goal is to make these metrics easy to understand over the original `rtcStatsReport`.

Reply:
Thanks for explaining!